include/slurp.h

```diff
@@ -53,6 +53,8 @@ struct slurp_state {
 	bool single_point;
 	bool restrict_selection;
 	struct wl_list boxes; // slurp_box::link
+	bool fixed_aspect;
```

Comment:
we could probably use an `aspect_ratio` of 0 to indicate no fixed aspect ration, since it doesn't really make sense to have an aspect ratio of 0

Reply:
But then, it is also good not having to think too much about corner cases. And using a 0.0 on a double may be annoying at some point.